packages/odyssey-react/src/components/Tag/Tag.module.scss

```diff
@@ -12,20 +12,20 @@
 
 .list {
   display: inline-block;
-  margin-block-end: -$spacing-xs;
-  margin-inline-start: $spacing-xs;
+  margin-block-end: calc(-1 * var(--ListSpaceMarginBlockEnd));
+  margin-inline-start: var(--ListSpaceMarginInlineStart);
   list-style-type: none;
 }
 
 .tag {
-  @include border-radius($base-border-radius);
+  @include border-radius(var(--TagBorderRadius));
```

Comment:
heh, this is an example of adding the "parts" as a prefix that is problematic. The component itself is already part of our API and named `Tag` so we then have duplication that would look like this:

```
theme={{
  [Tag.theme]: { TagBorderRadius: "..." }
}}
```

cc @andrewberg-okta @edburyenegren-okta 

My suggestion is to try renaming here to something else such as content?

Reply:
Agreed that Tag here feels awk. I could see this as either `.root` since it's the primary UI of the component. Or `.item` since it's the child of a list.

Reply:
Gotcha